infra/src/main/scala/Main.scala

```diff
@@ -0,0 +1,39 @@
+import besom.*
+import besom.api.kubernetes as k8s
+import besom.internal.Output
+import k8s.core.v1.{Service, namespace}
+import besom.internal.Config
+
+@main def main = Pulumi.run {
+  val conf = config
+  for
+    appNamespace <- namespace(name = "vss")
+    // zookeeper
+    zooDeployment <- Zookeeper.deploy(appNamespace)
```

Comment:
VSS: as mentioned above it would be nice if we can have an unified method to combine all of this calls.

Also, I wonder if the library can't collect exports so one do not need to manually define them?